utils/parsley/message_types.py

```diff
@@ -59,13 +62,15 @@
     "VENT" : 0x0B,
     "VENT_SPARE" : 0x0C,
     "GPS" : 0x0D,
-    "GPS_SPARE" : 0x0E
+    "GPS_SPARE" : 0x0E,
+    "FILL": 0x0F,
+    "FILL_SPARE": 0x010,
```

Comment:
this conflicts with the remote arming allocations, not yet merged though, we need to figure out who gets what. it is concerning that we are now out of board IDs, need to get on 29bit IDs I guess

Reply:
Remote arming isn't on canlib master and doesn't have a PR, so I think fill sensing should get first pick

Reply:
I'll leave it as is for now then